### PR TITLE
support load as null when meet 'null' string

### PR DESCRIPTION
**Pull request by jianwan0214**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue #5079 #4852 

## What this PR does / why we need it:
support load as null when meet 'null' string

### CODE REVIEW

**Review by iamlinjunhong — APPROVED** [review]

**Review by aunjgr — APPROVED** [review]

**Review by fengttt — COMMENTED** [review]
We want to support the CSV format options.  
See https://www.postgresql.org/docs/current/sql-copy.html    We want to support three delimiter, quote, and null.

Then there need to have bvt test on these 3 options.

**Comment by jianwan0214** [comment]
> Details



> We want to support the CSV format options. See https://www.postgresql.org/docs/current/sql-copy.html We want to support three delimiter, quote, and null.
> 
> Then there need to have bvt test on these 3 options.

ok

File: pkg/sql/plan/build_load.go [hunk 1]
```diff
@@ -168,3 +172,27 @@ func GetProjectNode(stmt *tree.Load, ctx CompilerContext, node *plan.Node, Name2
...
+
+func InitNullMap(stmt *tree.Load) error {
+	stmt.Param.NullMap = make(map[string][]string)
+	for i := 0; i < len(stmt.Param.Tail.Assignments); i++ {
+		expr, ok := stmt.Param.Tail.Assignments[i].Expr.(*tree.FuncExpr)
+		if !ok {
+			return moerr.NewError(moerr.INVALID_INPUT, "the load set list is not FuncExpr form")
+		}
+		if len(expr.Exprs) != 2 {
+			return moerr.NewError(moerr.INVALID_INPUT, "the nullif func need twp paramaters")
```
Comment (ouyuanning) [inline pkg/sql/plan/build_load.go:184]: typo?

Comment (jianwan0214) [reply pkg/sql/plan/build_load.go]: it should be two

**Review by ouyuanning — APPROVED** [review]

**Review by nnsgmsone — APPROVED** [review]

**Pull request merged**